mysql-test/suite/rocksdb/t/type_varchar.test

```diff
@@ -71,3 +71,14 @@ select 'email_i' as index_name, count(*) AS count from t force index(email_i);
 drop table t;
 set global rocksdb_checksums_pct = @save_rocksdb_checksums_pct;
 set session rocksdb_verify_row_debug_checksums = @save_rocksdb_verify_row_debug_checksums;
+
+# Issue #784 - Skip trailing space bytes for non-unpackable fields
+
+drop table if exists t;
```

Comment:
Remove

Reply:
Ahh, yeah, missed that (obviously). Will remove it, re-record and retest locally (skip jenkins) before merging. Thanks!